### PR TITLE
Added ability to post strategy concept weighting with percent

### PR DESCRIPTION
**Pull request by missinnale**
Addresses issue #86 to support concept weighting

### CODE REVIEW

File: terminalone/models/strategyconcept.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ class StrategyConcept(Entity):
         'strategy_id': int,
         'updated_on': t1types.strpt,
         'version': int,
+        'weighting': None,
```
Comment (pswaminathan) [inline terminalone/models/strategyconcept.py]: This is good, but `_push` object should have an enum for the field. See https://github.com/MediaMath/t1-python/blob/master/terminalone/models/strategy.py#L29 and https://github.com/MediaMath/t1-python/blob/master/terminalone/models/strategy.py#L92 for an example of defining it and putting it in.

File: terminalone/models/strategyconcept.py [hunk 2]
```diff
@@ -23,6 +23,8 @@ class StrategyConcept(Entity):
...
+        'percent': float,
```
Comment (pswaminathan) [inline terminalone/models/strategyconcept.py]: Keep this dict ordered alphabetically—move this to above "status".

**Comment by pswaminathan** [comment]
@missinnale can you allow me to make some changes here? Some nit-picky things. See https://github.com/blog/2247-improving-collaboration-with-forks.

**Comment by missinnale** [comment]
Yea, totally, apologies didn't see this comment until just now.

**Pull request merged**